filters/lang/filter.py

```diff
@@ -0,0 +1,24 @@
+from typing import List
+from interfaces.SentenceOperation import SentenceOperation
+from tasks.TaskTypes import TaskType
+from langid.langid import LanguageIdentifier, model
+
+
+class LanguageFilter(SentenceOperation):
+    tasks = [TaskType.TEXT_CLASSIFICATION, TaskType.TEXT_TO_TEXT_GENERATION]
+    languages = ["en"]
```

Comment:
This doesn't seem right. Can you add the list from the README here?

Reply:
Added in 5f720ee